Frameworks/UIKit/UIPresentationController.mm

```diff
@@ -19,24 +19,51 @@
 
 #include "UIKit/UIPresentationController.h"
 
-@implementation UIPresentationController
+@implementation UIPresentationController {
+    StrongId<UIViewController> _presentingViewController;
+    StrongId<UIViewController> _presentedViewController;
+    StrongId<UIView> _containerView;
+}
 
 /**
  @Status Interoperable
 */
 - (instancetype)initWithPresentedViewController:(UIViewController*)presentedViewController
              presentingViewController:(UIViewController*)presentingViewController {
-    _presentingViewController = presentingViewController;
-    _presentedViewController = presentedViewController;
-    _presentationStyle = _presentedViewController.modalPresentationStyle;
-    _containerView = _presentingViewController.view;
+    if (self = [super init]) {
+        _presentingViewController = presentingViewController;
```

Comment:
>_presentingViewController = presentingViewController [](start = 8, length = 52)

Is this right? Do we now managing the passed-in presentingViewController lifetime? If UIPresentationController is dealloc'ed, can we nil out the presentingViewController because we own it? Do we need to make a copy or something?